linkerd/http/access-log/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "linkerd-http-access-log"
-version = "0.1.0"
-authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
-license = "Apache-2.0"
+version = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
 edition = "2018"
-publish = false
+publish = { workspace = true }
```

Comment:
i've been intentional about this explicitly _not_ bumping any crates edition. that'd be an easy follow-on, but i want this change to be strictly a syntactic change.